pkg/mdformatter/linktransformer/link.go

```diff
@@ -48,9 +48,9 @@ func NewChain(c ...mdformatter.LinkTransformer) mdformatter.LinkTransformer {
 	return &chain{chain: c}
 }
 
-func (l *chain) TransformDestination(ctx mdformatter.SourceContext, destination []byte) (_ []byte, err error) {
+func (l *chain) TransformDestination(ctx mdformatter.SourceContext, destination []byte, lines string) (_ []byte, err error) {
```

Comment:
lines of what?  do you mean lineNumber?

I find it very confusing we need to pass lines through transformers args 🤔 

Why not adding it to `SourceContext`?

Reply:
Didn't think of that. Will try to implement!